rag_experiment_accelerator/doc_loader/pdfLoader.py

```diff
@@ -37,6 +40,28 @@ def preprocess_pdf_content(content: str):
     return content
 
 
+def _load_pdf_using_pypdf(file_path: str) -> List[Document]:
+    loader = PyPDFLoader(file_path=file_path)
+    return loader.load()
+
+
+def _load_pdf_using_unstructured(file_path: str) -> List[Document]:
```

Comment:
Severity: Potential Issue - The function '_load_pdf_using_unstructured' should also have a more descriptive name, such as 'load_pdf_with_unstructured'. This will enhance code readability.